lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
yes cursor blinks
I would just use damn sleep, but I know that @okurz would not like it